.gitignore

```diff
@@ -54,3 +54,11 @@ proto/tendermint/p2p/*.proto
 proto/tendermint/statesync/*.proto
 proto/tendermint/types/*.proto
 proto/tendermint/version/*.proto
+*.aux
+*.bbl
+*.blg
+*.log
+*.pdf
+*.gz
+*.dvi
+*.pb.go
```

Comment:
I'm pretty sure we do _not_ want to ignore `.pb.go` files, since we have them checked in all over the place to support the core types.

Reply:
Thanks for the spot. I'd just merged them together so hadn't done a thorough check but you're totally right.